tests/unit/backend/services/response.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
What do you think of moving this file to `tests/unit/support/fixtures/github-response.json`?

So `tests/unit/support/fixtures` will have anything that is not code (json or example files, mostly)

Reply:
Perfect!